scripts/scr_ui_settings/scr_ui_settings.gml

```diff
@@ -1,30 +1,42 @@
 function scr_ui_settings() {
 
+	var romanNumerals;
+	romanNumerals= scr_roman_numerals();
```

Comment:
next time please remember to declare and init on the same line, makes no sense to split it